fix(workstreams): $GSD_TOOLS env var undefined -- all workstream commands fail with module not found

## GSD Version

Latest (present in current `main`)

## Runtime

Multiple (affects all runtimes — the bug is in the command definition)

## Operating System

All (macOS, Windows, Linux)

## What happened?

The `/gsd-workstreams` command (`commands/gsd/workstreams.md`) references `$GSD_TOOLS` as the path to `gsd-tools.cjs` in all 6 of its subcommand invocations (`list`, `create`, `status`, `switch`, `progress`, `complete`, `resume`).

However, `GSD_TOOLS` is **never defined anywhere in the codebase** — it is not set by the installer, not exported by any shell hook, and not referenced by any other command file. All 60+ other workflow/command files use the resolved absolute path `$HOME/.claude/get-shit-done/bin/gsd-tools.cjs` instead.

Because `$GSD_TOOLS` resolves to an empty string, the LLM is forced to improvise the path at runtime, producing incorrect paths like `tools/gsd-tools.mjs` instead of `bin/gsd-tools.cjs`, causing a module not found error.

## What did you expect?

All workstream subcommands should resolve `gsd-tools.cjs` correctly, matching the pattern used by every other command in the codebase.

## Steps to reproduce

1. Install GSD with `npx get-shit-done-cc@latest`
2. Set up a project with `/gsd-init`
3. Create a workstream: `/gsd-workstreams` → `create`
4. The LLM attempts `node "$GSD_TOOLS" workstream create <name> ...` which fails because `$GSD_TOOLS` is empty

## Error output / logs

```
Error: Cannot find module '/Users/REDACTED/.claude/get-shit-done/tools/gsd-tools.mjs'
```

(The exact path may vary depending on how the LLM improvises the missing variable.)

## How often does this happen?

Every time (100% reproducible)

## Impact

Major — Core feature is broken. All workstream subcommands (`list`, `create`, `status`, `switch`, `progress`, `complete`, `resume`) are affected with no workaround other than manually editing the command definition.

## Workaround

Manually replace `$GSD_TOOLS` with the resolved path in `commands/gsd/workstreams.md`, or run the `gsd-tools.cjs` commands directly via the CLI.

## Additional context

This is the **exact same class of bug** as the one fixed in PR #1236 for `stats.md`, which was merged. That PR replaced the undefined `$GSD_TOOLS` in `stats.md` with the standard path. The same fix is needed here for `workstreams.md`.

**Scope:** Replace all 6 occurrences of `$GSD_TOOLS` in `commands/gsd/workstreams.md` with `$HOME/.claude/get-shit-done/bin/gsd-tools.cjs`.

**Note:** As mentioned in PR #1629, using the env var would be preferable for cross-platform support — but since it is currently undefined, the fix should first align with the established pattern (hardcoded path) to unblock users.

## Privacy Checklist

- [x] I have reviewed all pasted output for PII (usernames, paths, API keys) and redacted where necessary

--- a/commands/gsd/workstreams.md
+++ b/commands/gsd/workstreams.md
@@ -34,30 +34,30 @@ If no subcommand given, default to `list`.
 ## Step 2: Execute Operation
 
 ### list
-Run: `node "$GSD_TOOLS" workstream list --raw --cwd "$CWD"`
+Run: `node "$HOME/.claude/get-shit-done/bin/gsd-tools.cjs" workstream list --raw --cwd "$CWD"`
 Display the workstreams in a table format showing name, status, current phase, and progress.
 
 ### create
-Run: `node "$GSD_TOOLS" workstream create <name> --raw --cwd "$CWD"`
+Run: `node "$HOME/.claude/get-shit-done/bin/gsd-tools.cjs" workstream create <name> --raw --cwd "$CWD"`
 After creation, display the new workstream path and suggest next steps:
 - `/gsd-new-milestone --ws <name>` to set up the milestone
 
 ### status
-Run: `node "$GSD_TOOLS" workstream status <name> --raw --cwd "$CWD"`
+Run: `node "$HOME/.claude/get-shit-done/bin/gsd-tools.cjs" workstream status <name> --raw --cwd "$CWD"`
 Display detailed phase breakdown and state information.
 
 ### switch
-Run: `node "$GSD_TOOLS" workstream set <name> --raw --cwd "$CWD"`
+Run: `node "$HOME/.claude/get-shit-done/bin/gsd-tools.cjs" workstream set <name> --raw --cwd "$CWD"`
 Also set `GSD_WORKSTREAM` for the current session when the runtime supports it.
 If the runtime exposes a session identifier, GSD also stores the active workstream
 session-locally so concurrent sessions do not overwrite each other.
 
 ### progress
-Run: `node "$GSD_TOOLS" workstream progress --raw --cwd "$CWD"`
+Run: `node "$HOME/.claude/get-shit-done/bin/gsd-tools.cjs" workstream progress --raw --cwd "$CWD"`
 Display a progress overview across all workstreams.
 
 ### complete
-Run: `node "$GSD_TOOLS" workstream complete <name> --raw --cwd "$CWD"`
+Run: `node "$HOME/.claude/get-shit-done/bin/gsd-tools.cjs" workstream complete <name> --raw --cwd "$CWD"`
 Archive the workstream to milestones/.
 
 ### resume

--- a/tests/gsd-tools-path-refs.test.cjs
+++ b/tests/gsd-tools-path-refs.test.cjs
@@ -1,0 +1,57 @@
+/**
+ * Regression guard for #1766: $GSD_TOOLS env var undefined
+ *
+ * All command files must use the resolved path to gsd-tools.cjs
+ * ($HOME/.claude/get-shit-done/bin/gsd-tools.cjs), not the undefined
+ * $GSD_TOOLS variable. This test catches any command file that
+ * references the undefined variable.
+ */
+
+const { describe, test } = require('node:test');
+const assert = require('node:assert/strict');
+const fs = require('fs');
+const path = require('path');
+
+const COMMANDS_DIR = path.join(__dirname, '..', 'commands', 'gsd');
+
+describe('command files: gsd-tools path references (#1766)', () => {
+  test('no command file references undefined $GSD_TOOLS variable', () => {
+    const files = fs.readdirSync(COMMANDS_DIR).filter(f => f.endsWith('.md'));
+    const violations = [];
+
+    for (const file of files) {
+      const content = fs.readFileSync(path.join(COMMANDS_DIR, file), 'utf-8');
+      // Match $GSD_TOOLS or "$GSD_TOOLS" or ${GSD_TOOLS} used as a path
+      // (not as a documentation reference)
+      const lines = content.split('\n');
+      for (let i = 0; i < lines.length; i++) {
+        const line = lines[i];
+        if (/\$GSD_TOOLS\b/.test(line) && /node\s/.test(line)) {
+          violations.push(`${file}:${i + 1}: ${line.trim()}`);
+        }
+      }
+    }
+
+    assert.strictEqual(violations.length, 0,
+      'Command files must not reference undefined $GSD_TOOLS. ' +
+      'Use $HOME/.claude/get-shit-done/bin/gsd-tools.cjs instead.\n' +
+      'Violations:\n' + violations.join('\n'));
+  });
+
+  test('workstreams.md uses standard gsd-tools.cjs path', () => {
+    const content = fs.readFileSync(
+      path.join(COMMANDS_DIR, 'workstreams.md'), 'utf-8'
+    );
+    const nodeLines = content.split('\n').filter(l => /node\s/.test(l));
+
+    assert.ok(nodeLines.length > 0,
+      'workstreams.md should contain node invocations');
+
+    for (const line of nodeLines) {
+      assert.ok(
+        line.includes('gsd-tools.cjs'),
+        'Each node invocation must reference gsd-tools.cjs, got: ' + line.trim()
+      );
+    }
+  });
+});

